frontend/src/static/js/api/client.ts

```diff
@@ -256,4 +260,43 @@ export class APIClient {
 
     return allData;
   }
+
+  // Fetches feature counts for a browser in a date range
+  // via "/v1/stats/features/browsers/{browser}/feature_counts"
+  public async getFeatureCounts(
```

Comment:
Please use the AsyncIterable like in `getFeatureStatsByBrowserAndChannel`

This will allow us to draw the chart as data comes in.

Reply:
done